### DISCUSSION TITLE
Tadadadadadaaaa! Welcome to Discussions! 🚀

### DISCUSSION THREAD

**Discussion by fromITzerotoIThero** [fromITzerotoIThero/personal-website#7] [Announcements]
W., 

that looks like a perfect place for a conversation! Thanks for letting me know about it! 😉

**Comment by piotrwicijowski** [comment]
Glad you like it :)

**Comment by fromITzerotoIThero** [comment]
W., 
I was testing personal website on my phone and I've noticed that this isssue with navigating to `IBOSPage` (not displaying the very top of page) was still there. There was no horizontal scrollbar anymore, but the vertical one is there, though there's clearly enough space to fit the contents of the page. I must have missed why that happens when we talked about it. 
I guess I won't fix it before tomorrow, but it's just came up and I wanted to remember about it :)

**Reply by piotrwicijowski** [reply]
Hmm, so it would mean that there are two issues at play here - the one with rotated arrow was solved, but there is still something going on. I'll try to investigate

**Reply by piotrwicijowski** [reply]
I think I found a fix - I made a PR, please check if it fixes your problem - #16
